tests/utils/suite_writer/slash_run_result.py

```diff
@@ -111,7 +111,7 @@ def __getitem__(self, args):
 
     def __contains__(self, args):
         return self._normalize_key(args) in self._events_by_args
-        return self.has_event(*args)
+        # return self.has_event(*args)
```

Comment:
@vmalloc According to git `self.had_event(*args)` is the newer code of this method, unfortunately, it fails on unittests.  Is it ok to keep the previous version?  or should we fix the `has_event` method?